simple_netconf_client/__init__.py

```diff
@@ -34,3 +23,14 @@ def setup_logging(debug_enabled):
         logging_level = logging.INFO
 
     logging.basicConfig(level=logging_level, format='%(asctime)s - %(levelname)s - %(message)s')
+
+try:
+    version_file_path = resource_path('version.txt')
+    with open(version_file_path) as f:
+        __version__ = f.readlines()[0].strip()
+except Exception:
+    # package is not installed
+    __version__ = "0.0.0"
```

Comment:
Running the program from a `git clone` with `python main.py` now shows version '0.0.0'.  I would've expected 'v2.0.0-8-g6de3873' == 8 commits after tag 'v2.0.0', specifically git has 6de3873.

Tricky.  We don't need 'import git' at runtime for people running the .exe.  Maybe we can keep the code in `__init__.py` and only call it if we don't have `version.txt` perhaps?

Reply:
I've pushed new commit addressing this issue. You can review.

Reply:
Thanks, looks good to me! :sunglasses::+1: 

Appreciate you remember updating `requirements.txt` as well :pray: